guide/packages.md

```diff
@@ -0,0 +1,155 @@
+# Membrane packages
+
+> If you see any out-of-date information or an invalid link here, feel free to create an issue or open a PR in [the guide repository](https://github.com/membraneframework/guide)
+
+To improve modularity, Membrane is split into multiple packages. Here we gathered all of the official ones and keep adding third-party ones too, so if you created a Membrane plugin or other Membrane-based library, let us know!
+We use the following [tags] in package descriptions:
+- [In progress] - in active development, but not yet ready
+- [Experimental] - possible to use, but may lack some core features/tests/documentation
+- [Alpha] - ready to use, but may lack some features or need more testing
+- [Third party] - maintained by the community
+- [Suspended] - not ready and development was suspended
+
+Check the [Membrane GitHub organization](https://github.com/membraneframework) to see which packages are actively developed currently.
+
+## Demos
+All the official Membrane demos reside in the [membrane-demo](https://github.com/membraneframework/membrane-demo) package. They are:
+- [Basic demos](https://github.com/membraneframework/membrane-demo/tree/master/basic)
+- [RTP demo](https://github.com/membraneframework/membrane-demo/tree/master/rtp)
+- [Receiving RTP stream and publishing it via HLS](https://github.com/membraneframework/membrane-demo/tree/master/rtp)
+- [WebRTC signaling server demo](https://github.com/membraneframework/membrane-demo/tree/master/webrtc)
+
+## Docker
+
+The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [GitHub](https://github.com/membraneframework/docker-membrane)) contains Erlang, Elixir and libraries necessary to test and run the Membrane Framework.
+
+## Media agnostic packages
+
+| Package           | Description                                                                     | Links                                                                                                                                                                    |
+| ----------------- | ------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| membrane_core     | The core of the framework                                                       | [Hex](https://hex.pm/packages/membrane-core)&nbsp;[Docs](https://hexdocs.pm/membrane-core)&nbsp;[GitHub](https://github.com/membraneframework/membrane-core)             |
```

Comment:
Almost all the Hex and HexDocs links are broken (packages have underscores, not hyphens)